pinot-spi/src/test/java/org/apache/pinot/spi/plugin/PluginManagerTest.java

```diff
@@ -96,19 +96,19 @@ public void testGetPluginsToLoad()
     HashMap<String, File> actualPluginsMap = PluginManager.get().getPluginsToLoad(pluginsDirs, pluginsToInclude);
     Assert.assertEquals(actualPluginsMap.size(), 3);
 
-    ArrayList<String> actualPluginNames = new ArrayList<>();
-    ArrayList<String> actualPluginPaths = new ArrayList<>();
+    HashSet<String> actualPluginNames = new HashSet<>();
```

Comment:
Let's make it a string to string map and compare the map instead of comparing 2 sets

Reply:
Thanks for the advice, updated.